docs/standard/tour.md

```diff
@@ -44,129 +42,79 @@ Microsoft languages that .NET supports include C#, F#, and Visual Basic.
 
 ## Automatic memory management
 
-Garbage collection is the most well-known of .NET features. Developers do not need to actively manage memory, although there are mechanisms to provide more information to the garbage collector (GC). C# includes the `new` keyword to allocate memory in terms of a particular type, and the `using` keyword to provide scope for the usage of the object. The GC operates on a lazy approach to memory management, preferring application throughput to the immediate collection of memory.
+.NET uses [garbage collection](garbagecollection/index.md) to provide automatic memory management for programs.  The GC operates on a lazy approach to memory management, preferring application throughput to the immediate collection of memory.  To learn more about the .NET GC, check out [Fundamentals of garbage collection](garbagecollection/fundamentals.md).
```

Comment:
garbage collection -> garbage collection (GC)

I'd also avoid the verb check out and use see or visit instead.